release-notes.md

```diff
@@ -1,3 +1,1790 @@
+- multi: Use latest base58 module ([decred/dcrd#2223](https://github.com/decred/dcrd/pull/2223))
+- standalone: Start v2 module dev cycle ([decred/dcrd#2224](https://github.com/decred/dcrd/pull/2224))
+- multi: go mod tidy cleanup and run in CI ([decred/dcrd#2225](https://github.com/decred/dcrd/pull/2225))
+- mempool: Move to internal ([decred/dcrd#2274](https://github.com/decred/dcrd/pull/2274))
+- mining: Move to internal ([decred/dcrd#2275](https://github.com/decred/dcrd/pull/2275))
+- rpcserver: Move to internal ([decred/dcrd#2288](https://github.com/decred/dcrd/pull/2288))
+- fees: Move to internal ([decred/dcrd#2287](https://github.com/decred/dcrd/pull/2287))
+- main: go mod tidy ([decred/dcrd#2367](https://github.com/decred/dcrd/pull/2367))
+- dcrjson: Prepare v3.1.0 ([decred/dcrd#2374](https://github.com/decred/dcrd/pull/2374))
+- addrmgr: Prepare v1.2.0 ([decred/dcrd#2375](https://github.com/decred/dcrd/pull/2375))
+- connmgr: Prepare v3.0.0 ([decred/dcrd#2376](https://github.com/decred/dcrd/pull/2376))
+- multi: Update chaincfg dependers to wire/v1.4.0 ([decred/dcrd#2381](https://github.com/decred/dcrd/pull/2381))
+- chaincfg: Prepare v3.0.0 ([decred/dcrd#2381](https://github.com/decred/dcrd/pull/2381))
+- dcrutil: Prepare v3.0.0 ([decred/dcrd#2383](https://github.com/decred/dcrd/pull/2383))
+- rpc/jsonrpc/types: Prepare v2.1.0 ([decred/dcrd#2385](https://github.com/decred/dcrd/pull/2385))
+- txscript: Prepare v3.0.0 ([decred/dcrd#2384](https://github.com/decred/dcrd/pull/2384))
+- blockchain: Update unreleased requires to master ([decred/dcrd#2364](https://github.com/decred/dcrd/pull/2364))
+- rpcclient: Update unreleased requires to master ([decred/dcrd#2369](https://github.com/decred/dcrd/pull/2369))
+- blockchain/standalone: Remove txscript dep ([decred/dcrd#2388](https://github.com/decred/dcrd/pull/2388))
+- database: Prepare v2.0.2 ([decred/dcrd#2387](https://github.com/decred/dcrd/pull/2387))
+- hdkeycahin: Prepare v3.0.0 ([decred/dcrd#2392](https://github.com/decred/dcrd/pull/2392))
+- blockchain/standalone: Prepare v2.0.0 ([decred/dcrd#2395](https://github.com/decred/dcrd/pull/2395))
+- blockchain/stake: Prepare v3.0.0 ([decred/dcrd#2418](https://github.com/decred/dcrd/pull/2418))
+- gcs: Prepare v2.1.0 ([decred/dcrd#2420](https://github.com/decred/dcrd/pull/2420))
+- peer: Prepare v2.2.0 ([decred/dcrd#2422](https://github.com/decred/dcrd/pull/2422))
+- rpcclient: Prepare v6.0.0 ([decred/dcrd#2423](https://github.com/decred/dcrd/pull/2423))
+- blockchain: Prepare v3.0.0 ([decred/dcrd#2424](https://github.com/decred/dcrd/pull/2424))
+- rpcclient: Prepare v6.0.1 ([decred/dcrd#2455](https://github.com/decred/dcrd/pull/2455))
+- main: Update to use all new module versions ([decred/dcrd#2426](https://github.com/decred/dcrd/pull/2426))
+- main: Remove module replacements ([decred/dcrd#2428](https://github.com/decred/dcrd/pull/2428))
+- main: Use backported module up
```

Comment:
dcrd release notes got truncated. Two commit listings and Code Contributors sections are now missing.

Reply:
Fix to restore the truncated dcrd section pushed alogn other tweaks in https://github.com/alexlyp/decred-binaries/pull/1